File: clas.py
class Point:
    def __init__(self, x=0, y=0):
        self.x = x
        self.y = y

    def get_x(self):
        return self.x

    def set_x(self, value):
        self.x = value

    def get_y(self):
        return self.y

    def set_y(self, value):
        self.y = value

    def distance(self, point):
        return ((self.x - point.x) ** 2 + (self.y - point.y) ** 2) ** 0.5

    def norm(self):
        return (self.x ** 2 + self.y ** 2) ** 0.5

# Class for testing
class TestPoint:
    def test(self):
        point1 = Point(5,7)
        point2 = Point(8,6)

        print(f"Default point coordinates: ({point1.get_x()}, {point1.get_y()})")
        print(f"Point2 coordinates: ({point2.get_x()}, {point2.get_y()})")
        print(f"Distance between default point and Point2: {point1.distance(point2)}")
        print(f"Norm of Point2: {point2.norm()}")

# Create an instance of TestPoint and execute the test
test_point = TestPoint()
test_point.test()

File: clas.java
public class Point {
    private double x;
    private double y;

    public Point() {
        this.x = 0;
        this.y = 0;
    }

    public Point(double x, double y) {
        this.x = x;
        this.y =y;
    }

    public double get_x() {
        return x;
    }

    public void set_x(double valeur) {
        this.x = valeur;
    }

    public double y() {
        return y;
    }

    public void set_y(double valeur) {
        this.y = valeur;
    }

    public double distance(Point point) {
        return Math.sqrt(Math.pow(this.x - point.x, 2) + Math.pow(this.y - point.y, 2));
    }

    public double norm() {
        return Math.sqrt(Math.pow(this.x, 2) + Math.pow(this.y, 2));
    }

    public static void main(String[] args) {
        Point point1 = new Point(4, 3);

        System.out.println("Distance: " + point1.distance(new Point(0, 0)));
        System.out.println("Norm: " + point1.norm());
    }
}